hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieInternalConfig.java

```diff
@@ -31,11 +31,21 @@ public class HoodieInternalConfig extends HoodieConfig {
   public static final String BULKINSERT_ARE_PARTITIONER_RECORDS_SORTED = "hoodie.bulkinsert.are.partitioner.records.sorted";
   public static final Boolean DEFAULT_BULKINSERT_ARE_PARTITIONER_RECORDS_SORTED = false;
 
+  public static final String BULK_INSERT_OVERWRITE_TABLE = "INSERT_OVERWRITE_TABLE";
+  public static final String BULK_INSERT_OVERWRITE = "INSERT_OVERWRITE";
+
   public static final ConfigProperty<String> BULKINSERT_INPUT_DATA_SCHEMA_DDL = ConfigProperty
       .key("hoodie.bulkinsert.schema.ddl")
       .noDefaultValue()
       .withDocumentation("Schema set for row writer/bulk insert.");
 
+  public static final ConfigProperty<String> BULKINSERT_OVERWRITE_MODE = ConfigProperty
+      .key("hoodie.bulkinsert.overwrite.mode")
```

Comment:
The value for this config is a write operation type. So, its key should be named accordingly.